### DISCUSSION TITLE
Reset wifi credentials

### DISCUSSION THREAD

**Discussion by Smiggy1981** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#354] [Q&A]
Hi there, 

Firstly great work on this project the instructions where easy to follow, managed to get a few printed boards and have been using one of them for over a month  now.

However I have just had to swap my network router and  my lay z spa is still trying to connect with the old details. Is there any easy way to swap the network details without pulling the node out of the unit? I seem to recall reading about a button press sequence although I could be wrong because despite searching I was unable to find this again.

Any help would be very much appreciated.

Thanks 

Smiggy

**Comment by visualapproach** [comment]
Hi, yes the button sequence is described in the build instructions. It will erase the Wi-Fi credentials and restart to the AP where you can enter new creds.

**Reply by visualapproach** [reply]
There is also a url layzspa.local/resetwifi/ if I remember correctly. If you have a 4-wire that would be the only option without removing the device

**Reply by Smiggy1981** [reply]
Thanks for the quick response, I'll take another look at the build instructions. Apologies I did have a quick run through knowing I had read it some where but must have missed it
